src/Services/services.js

```diff
@@ -0,0 +1,9 @@
+import axios from 'axios';
+
+export const GetAllVideos = async () => {
+  return axios.get('/api/videos');
+};
+
+export const GetAllCategories = async () => {
+  return axios.get('/api/categories');
+};
```

Comment:
Returning a promise not value
Check once if this is what is intended

Reply:
it is intentional, I'm returing a promise only here